app/scripts/directives/blocks.js

```diff
@@ -234,3 +246,104 @@ angular
       },
     };
   });
+
+angular
+  .module('confRegistrationWebApp')
+  .directive('opportunitiesQuestion', function () {
+    return {
+      templateUrl: opportunitiesQuestionTemplate,
+      restrict: 'E',
+      controller: function ($scope) {
+        $scope.block.content.staticChoices = [
+          'Yes, via email',
+          'Yes, via text',
+          'Yes, via email & text',
+          'No',
+        ];
+      },
+    };
+  });
+
+angular
+  .module('confRegistrationWebApp')
+  .directive('graduationDateQuestion', function () {
+    return {
+      templateUrl: graduationDateQuestionTemplate,
+      restrict: 'E',
+    };
+  });
+
+angular
+  .module('confRegistrationWebApp')
+  .directive('ethnicityQuestion', function () {
+    return {
+      templateUrl: ethnicityQuestionTemplate,
+      restrict: 'E',
+      controller: function ($scope, $timeout) {
+        $scope.otherSentinel = '__other__';
+
+        $scope.block.content.staticChoices = [
+          'American Indian/Native Alaskan',
```

Comment:
I think they prefer `Native American` but I would double check.

Reply:
This was the official list they got from HR.

Reply:
ah okay use that then